bin/auth0_create.js

```diff
@@ -132,7 +129,7 @@ function handleCreate(args) {
             action: 'created', 
             onOutput: function (log, build, url) {
                 var action = claims ? 'updated' : 'created';
-                var state = args.meta['auth0-extension-disabled'] ? 'disabled' : 'enabled';
+                var state = args.meta['auth0-extension-disabled'] === "1" ? 'disabled' : 'enabled';
```

Comment:
Might be worth linking to the other side of the 'contract': https://github.com/auth0/auth0-ext/blob/b88d38546785cd326f45b4c36e676d240c3b7527/lib/executeExtension2.js#L751-L753